### ISSUE TITLE
[paper] curl: (22) 404 when trying to download newest version

### ISSUE THREAD

**Issue by KS-HTK** [itzg/docker-minecraft-server#916]
When running PAPER there seems to be a error in detecting the newest release.
Today i got this error.
```
[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x   12 1000     1000          4096 Jun  9 00:43 /data'
[init] Resolved version given LATEST into 1.17
[init] Resolving type given paper
curl: (22) The requested URL returned error: 404
```

There is no release for version 1.17 of PaperMC yet.
What am I missing?

**Comment by ChrisKraw** [comment]
I've got the same issue.

**Comment by itzg** [comment]
You'll need to set `VERSION` to something specific until Paper releases for 1.17

**Comment by KS-HTK** [comment]
Ok, it would be nice (if possible) to have the newest version check run for the selected Servertype and not just newest Vanilla version.
Would be a nice enhancement, but is certainly no critical requirement.

**Comment by itzg** [comment]
I checked the code and falling back to the latest version they have published is already supported. Make sure to pull the latest docker image because my logs show:

```
[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 2 1000 1000 4096 Jun  9 12:36 /data'
[init] Resolved version given LATEST into 1.17
[init] Resolving type given PAPER
curl: (22) The requested URL returned error: 404 
[init] WARN: using 1.16.5 since that's the latest provided by PaperMC
[init] Removing old PaperMC versions ...
[init] Downloading PaperMC 1.16.5 (build 777) ...
```

**Comment by KS-HTK** [comment]
It launched the server if it had been downloaded.
I just tested it, and it is working now. I had updated 3 days ago. So that may have solved it.
Thanks
